### ISSUE TITLE
Bitmap as continent input

### ISSUE THREAD

**Issue by irbian** [VPJPeltonen/Godot-WorldMaker#3]
Hi!

Would be an option that, instead of random continents it would receive a bitmap as base input to determine the continents?

**Comment by VPJPeltonen** [comment]
I have been thinking about doing something like that. Would the idea then be generate climates and civs on that and skip the landmass generation?

**Comment by irbian** [comment]
Yeah. Lots of map generators have this elevation and weather generators but I have yet to found something that, provided a drawn map for the user, generate the rest.

**Comment by VPJPeltonen** [comment]
I'll at least look into it next whenever I get around to updating. It would be interesting feature to add
